packages/suite/src/utils/suite/distribution.ts

```diff
@@ -0,0 +1,64 @@
+import { createHash } from 'crypto';
+
+import { Groups, Message } from '@suite-common/suite-types';
+import { RequiredKey } from '@trezor/type-utils';
+
+interface DistributionMessageProps extends RequiredKey<Message, 'groups'> {}
+
+interface DistributionCategoriesProps {
+    distribution: DistributionMessageProps | undefined;
+    instanceId: string | undefined;
+}
+
+interface DistributionGetGroupByInclusion {
+    groups: DistributionMessageProps['groups'];
+    inclusion: number;
+}
+
+/**
+ * @returns number between 0 and 99 generated from instanceId
+ */
+export const getInclusionFromInstanceId = (instanceId: string) => {
```

Comment:
The numbers 0 to 95 are slightly more likely to occur than numbers 96 to 99. 🤓 

https://chatgpt.com/share/672e3087-1564-800b-8479-39ce0b483749

But practically, for our use, it should be completely negligible 👌